docs/intro/ff-ui/toolbar.md

```diff
@@ -0,0 +1,249 @@
+
+:::
+
+![Optimization.avif](imgs/Optimization.avif)
+
+## 9. Project issues
+
+If there are any issues or warnings present in your current project that might result in a build failure or app crash, they will be displayed in this section. You can click this option to view a short description of each issue and navigate to the place where you can fix the issue by clicking on the respective issue.
+
+Errors prevent your app from compiling and running. These must be resolved in order to run the app. They can be due to missing actions, errors in custom code, incorrect data types, and so on.
+
+Warnings, while not preventing compilation, indicate potential issues such as incorrect rules setup or performance problems. Although it's possible to ignore warnings, addressing them can enhance the quality of your app and prevent future issues.
+
+![Warning-and-errors.avif](imgs/Warning-and-errors.avif)
+
+## 10. Branching
+
+[Branching](#) allows you to create a separate copy of your existing project to build new things without breaking existing features.
+
+## 11. Developer menu
+
+From here, you can view the code for your project, connect to GitHub, and also download the source code for your project.
+
+### 11.1 View Code
+
+This option lets you display the *Dart* code for all the pages of your FlutterFlow project. You can also take a look at the dependencies being used by the app here.
+
+### 11.2 Connect GitHub Repo (Paid Feature)
+
+You can use this option to connect your project with your [GitHub](https://github.com/) account and upload it to a GitHub repository. When you click this option, you will be navigated to the Settings and Integrations > Integrations > Github section.
+
+From there, you can configure your project to add GitHub integration to it. You can get a detailed [step-by-step guide](#) for connecting to GitHub.
+
+### 11.3 Download Code
+
+You can download the entire codebase of the app generated by FlutterFlow using this option. It will download a `.zip` file. You can open the contents using any code editor to make modifications.
+
+:::note
+This feature requires a paid plan.
+:::
+
+### 11.4 Download APK
+
+Use this to generate a release build of your app. It will automatically download the `.apk` file after the building process is complete.
+
+:::note
+This feature requires a paid plan.
+:::
+
+### 11.5 FlutterFlow CLI
+
+You can also download the code using *[FlutterFlow CLI](https://pub.dev/packages/flutterflow_cli)*. See instructions [here](#).
+
+## 12. Share project
+
+You can make a project public so that others can view and clone your project. Before you share your project, make sure to remove any sensitive information.
+
+:::note
+
+- You can only share projects where you are the owner.
+- The share feature can be used to create Marketplace items. You can learn more about this [here](https://docs.flutterflow.io/marketplace/adding-and-purchasing-items#adding-a-marketplace-item).
+
+:::
+
+## 13. Preview
```

Comment:
I'm wondering if this is becoming repetitive because John explains this and testing  in How to Run & test projects. Can we simply link it there and reduce the following headers to 1 header and just link to the other page?

Reply:
Ok, updated!